From f49dcd9282dcac20a089dc4172d6485a939f2d41 Mon Sep 17 00:00:00 2001
From: Zhixiong Chi <zhixiong.chi@windriver.com>
Date: Mon, 13 Apr 2020 03:34:49 -0700
Subject: [PATCH 4/5] MLK-21149-02 arm64: dts: imx: fix the build warning of
 endpoint on imx8mm

commit 14ad4182cf6fd83d6e13034e3d053999d72d6a4a from
https://source.codeaurora.org/external/imx/linux-imx.git

Fix the build warning of endpoint to follow the device tree spec.

Signed-off-by: Jacky Bai <ping.bai@nxp.com>
Signed-off-by: Zhixiong Chi <zhixiong.chi@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mm-evk.dts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk.dts
index b548f4d2974c..5e0fc6349d14 100755
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk.dts
@@ -727,7 +727,7 @@
 	#size-cells = <0>;
 	status = "okay";
 	port {
-		mipi1_sensor_ep: endpoint1 {
+		mipi1_sensor_ep: endpoint@1 {
 			remote-endpoint = <&ov5640_mipi1_ep>;
 			data-lanes = <2>;
 			csis-hs-settle = <13>;
@@ -735,7 +735,7 @@
 			csis-wclk;
 		};
 
-		csi1_mipi_ep: endpoint2 {
+		csi1_mipi_ep: endpoint@2 {
 			remote-endpoint = <&csi1_ep>;
 		};
 	};
-- 
2.17.1

